examples/torchrec_dlrm/README.md

```diff
@@ -0,0 +1,45 @@
+
+### TorchRec DLRM Example
+
+This example shows how to serve a Deep Learning Recommendation Model (DLRM) with TorchRec on a single GPU (CPU is currently not yet supported by this example). It requres at least 15GB of free GPU memory to run.
+The DLRM is an open source model for personalization and recommendation use cases published by Meta. More informatino can be found in this [paper](https://arxiv.org/abs/1906.00091) and this [blog post](https://ai.facebook.com/blog/dlrm-an-advanced-open-source-deep-learning-recommendation-model/).
+TorchRec is Meta's open source library for recommender systems in Pytorch. More information on TorchRec can be found in the [official docs](https://pytorch.org/torchrec/).
+
+--- **This example requires CUDA version >= 11.3**  ---
+
+In this example we will first create and archive the DLRM into a mar file which is subsequently registered in a TorchService instance. Finally, we run an inferent using curl.
+
+To create the model and archive it we simple need to run
+
```

Comment:
can we add instruction/ note to install torchrec?

Reply:
This has been added under requirements/common.txt 
What is the right strategy here? This would apply to any other domain library torchserve wants to integrate with? @msaroufim @HamidShojanazeri

Reply:
Added instruction in readme. Removed from common.txt